packages/pds-ember/app/styles/pds/core/typography/_config.scss

```diff
@@ -0,0 +1,87 @@
+$_module: "PDS.Core.Typography";
+/* --- [debug] CONFIG: #{$_module} --- */
+@use "../../theme";
+
+// Mixins defined per specifications in Figma (Structure 1.9 > Text Styles)
+
+@mixin Heading($level: 2) {
+  /* [debug] #{$_module}@Heading */
+
+  font-weight: theme.$heading-fontWeight;
+  line-height: theme.$heading-lineHeight;
+
+  @if $level == 0 {
```

Comment:
Is this 0 just to denote that it's bigger than a normal H1?

Reply:
This is the level designation as defined in Figma.